packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/example.html

```diff
@@ -8,16 +8,22 @@ <h3>Default Zones</h3>
   <gux-time-zone-picker-beta
     value="Etc/GMT+1"
     workspace-default="Etc/GMT"
-    ,
     local-default="America/Detroit"
```

Comment:
Do we have any validation on these strings to check if they are really timezones?